includes/shortcodes/arlo-upcoming-events.php

```diff
@@ -293,7 +293,7 @@ private static function generate_list_sql($atts, $import_id, $for_pagination = f
 
         $join = [];
         // We check that the event startdatetime with offset (converted to UTC) has not passed
```

Comment:
Could we remove this comment about startdatetime if we aren't filtering for it anymore?

Reply:
It's been removed :)